src/Components/PaperEdits/PaperEdit/ProgramScript/index.js

```diff
@@ -488,20 +484,17 @@ class ProgramScript extends Component {
     }
   }
 
+  // information around progressbar in the playlist object
   render() {
     return (
       <Tab.Content>
         <h2
           className={ [ 'text-truncate', 'text-muted' ].join(' ') }
           title={ `Programme Script Title: ${ this.state.programmeScript ? this.state.programmeScript.title : '' }` }>
-          {/* Programme:  */}
-          {/* <small> */}
           {this.state.programmeScript ? this.state.programmeScript.title : ''}
-          {/* </small> */}
         </h2>
-        {/* <hr/> */}
         { !this.state.resetPreview ?
-          <PreviewCanvas playlist={ this.state.playlist } width={ '300' }/>
+          <PreviewCanvas playlist={ this.state.playlist } programmeScript={ this.state.programmeScript } />
```

Comment:
This is nice :)

Reply:
yeah didn't want a hardcoded width thing :)